### ISSUE TITLE
Client is using unmaintained + deprecated packages. 

### ISSUE THREAD

**Issue by pikiaboy** [bitpay/nodejs-bitpay-client#49]
**Setup** 

- New node project
- yarn add --dev bitpay-sdk

Note the warnings of deprecated and unmaintained packages being used. 

```
warning bitpay-sdk > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning bitpay-sdk > request-promise@4.2.6: request-promise has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
warning bitpay-sdk > request > har-validator@5.1.5: this library is no longer supported
warning bitpay-sdk > request > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

**Comment by anbugal** [comment]
We're currently working on this, it will be updated in the version 3.0

**Comment by anbugal** [comment]
Fixed on v3.0.2202
